terra_layer/models.py

```diff
@@ -47,14 +87,14 @@ class LayerGroup(models.Model):
 
     class Meta:
         unique_together = ["view", "label", "parent"]
-        ordering = ["order"]
+        ordering = ["order", "label"]
```

Comment:
Really necessary? Order is set by `tree2models()` already

Reply:
It can't hurt.

Reply:
It really is not necessary, and as the groups will eventually be dropped, no need to add noise in this PR. Also it hurts :)

> Ordering is not a free operation. Each field you add to the ordering incurs a cost to your database. Each foreign key you add will implicitly include all of its default orderings as well.

from https://docs.djangoproject.com/en/2.2/ref/models/options/#django.db.models.Options.ordering

I'm okay with the one from `Layer` though

Reply:
Ok ;-) I remove the one in layer group